internal/cassandra/cassandra_client.go

```diff
@@ -9,16 +9,19 @@ import (
 
 var _ Client = &cassandraClient{}
 
+// CassandraClient model for transactions with cassandra
```

Comment:
Not sure if we really need to have documentation on private level structs. I think that we can remove this comment